draft-mouris-cfrg-mastic.md

```diff
@@ -270,6 +301,10 @@ MUST be in range `[0, 2^16 - 11)`.
 | `BITS: int`       | bit length of the input string `alpha`                | set by constructor                                         |
 | `VALUE_LEN: int`  | length of `beta`                                      | set by constructor                                         |
 | `field: type[F]`  | class object for the field ({{Section 6.1 of !VDAF}}) | set by constructor                                         |
+| `Field.MODULUS: int`   | prime modulus defining the field arithmetic           | to be defined                                              |
+| `MEAS_LEN: int`        | the length of the encoded measurement                 | to be defined                                              |
+| `OUTPUT_LEN: int`      | the length of the aggregatable output              | to be defined                                              |
+| `SEED_SIZE: int`       | the size (in bytes) of the XOR seed                   | 32 for `XofTurboShake128` or 16 for `XofFixedKeyAes128`
```

Comment:
I'm not quite sure why `MEAS_LEN` and `OUTPUT_LEN` are relevant to VIDPF. Also, why add `Field.MODULUS` here?

Reply:
We mention MEAS_LEN on L725 and OUTPUT_LEN on L1175. Since they are mentioned in the draft text, we need to define. Where are we defining field.MODULUS?

Reply:
`MEAS_LEN` and `OUTPUT_LEN` are parameters of `flp`, not of `vidpf`, which is why they're not relevant here.

Reply:
I think this depends on how we want to reference them; someone reading the draft should either see a pointer, or a definition. These are also defined here: https://github.com/cfrg/draft-irtf-cfrg-vdaf/blob/bd983637a87a7fa8353db5771c184585b2550271/draft-irtf-cfrg-vdaf.md?plain=1#L2520

Reply:
I guess I'm not clear on what we're trying to accomplish. Currently this table lists the parameters of VIDPF; if we also want to have a table of FLP parameters, then we either need to change the caption of this table or create a new table elsewhere.

There is already a paragraph in {{conventions}} in our draft with a rather terse description of FLP syntax. Perhaps we should just expand on that?

Reply:
Works for me.